test/ClangImporter/pcm-emit-direct-cc1-mode.swift

```diff
@@ -9,7 +9,7 @@
 // CHECK-DUMP:   Module map file: {{.*[/\\]}}Inputs{{/|\\}}custom-modules{{/|\\}}module.modulemap
 
 // Verify that the clang command-line used is cc1
-// RUN: %FileCheck -check-prefix CHECK-CLANG -DTRIPLE=%target-triple %s < %t.diags.txt
+// RUN: %FileCheck -check-prefix CHECK-CLANG -DTRIPLE=%module-target-triple %s < %t.diags.txt
```

Comment:
The issue is that the target triple is `armv7-unknown-linux-androideabi`, while the Swift compiler passes along the slightly different module triple `armv7-unknown-linux-android` to the ClangImporter. I think some other platforms have similar differences, so I don't know why this would be failing on Android armv7 alone, unless it really is the only one on the official CI that varies like this.

Reply:
This fix failed on macOS, as it passes the full `x86_64-apple-macosx13.0` triple to the ClangImporter, not the abbreviated module triple `x86_64-apple-macos` like Android armv7 does. Since these platforms are doing different things, simply disabled this test for Android armv7 for now.